common/components/AddressFieldFactory/AddressFieldDropdown.scss

```diff
@@ -0,0 +1,42 @@
+@import 'common/sass/variables';
+@import 'common/sass/variables/colors';
+@import 'common/sass/variables/spacing';
+
+.AddressFieldDropdown {
+  position: absolute;
+  top: calc(100% - 16px);
+  border: 1px solid $gray-lighter;
+  border-top: none;
+  background: $component-active-color;
+  z-index: 1;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+
+  &-dropdown-item {
+    margin: 0;
+    padding: $space-md $space;
+    display: flex;
+    align-items: center;
+
+    &:hover {
+      cursor: pointer;
+      background: $gray-lightest;
+    }
+
+    strong:first-of-type {
+      width: $space-xl * 6;
```

Comment:
Rather than give labels a fixed width, which can look weird if they're all short or you only have one, I'd suggest doing

(Identicon) **Label** _address_

So that it can just be left aligned as normal.

Reply:
:+1: